doc/nrf/external_comp/memfault.rst

```diff
@@ -153,10 +160,8 @@ The **Coredumps** section of the following sample documents provides more inform
 See `Memfault: Coredumps`_ for detailed documentation.
 Faults are captured and displayed on the **Issues** page of the Memfault UI.
 
-.. _ug_memfault_metrics:
-
 Collecting Metrics
-==================
+******************
```

Comment:
Line 163: Collecting metrics??

Reply:
This needs to be checked.

Reply:
Checked with Jamie and Collecting metrics should be used. Will update the docs, wherever applicable.